ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource

```diff
@@ -22,29 +22,24 @@ ${DC_WORKBENCH_SELECTOR_XP}=         xpath=//div[contains(@class,"modal")]//div[
 *** Keywords ***
 Create S3 Data Connection
     [Documentation]    Creates a S3 Data Connection from DS Project details page
-    [Arguments]    ${project_title}    ${dc_name}    ${aws_access_key}    ${aws_secret_access}
+    [Arguments]    ${project_title}    ${dc_name}    ${aws_access_key}    ${aws_secret_access}    ${workbench_title}=${NONE}
     ...            ${aws_s3_endpoint}=${S3.AWS_DEFAULT_ENDPOINT}    ${aws_region}=${S3.AWS_DEFAULT_REGION}
     ...            ${connected_workbench}=${NONE}    ${press_cancel}=${FALSE}    ${aws_bucket_name}=${NONE}
-    Click Button    ${DC_ADD_BTN_1_XP} 
+    Click Button    ${DC_ADD_BTN_1_XP}
     Wait Until Page Contains Element    ${S3_NAME_DC_INPUT_XP}
     Input Text    ${S3_NAME_DC_INPUT_XP}    ${dc_name}
     Input Text    ${S3_KEY_DC_INPUT_XP}    ${aws_access_key}
     Input Text    ${S3_SECRET_DC_INPUT_XP}    ${aws_secret_access}
     Input Text    ${S3_ENDPOINT_DC_INPUT_XP}    ${aws_s3_endpoint}
     Input Text    ${S3_REGION_DC_INPUT_XP}    ${aws_region}
-    ${version_check}=  Is RHODS Version Greater Or Equal Than  1.20.0
-    IF  ${version_check}==True
-        Input Text    ${S3_BUCKET_DC_INPUT_XP}    ${aws_bucket_name}
-    END
-    IF    "${connected_workbench}" == "${NONE}"
+    Input Text    ${S3_BUCKET_DC_INPUT_XP}    ${aws_bucket_name}
+    IF  "${connected_workbench}" == "${NONE}"
         Log    msg=you are not connecting any workbenchs to ${dc_name} DataConnection
     ELSE
         Run Keyword And Continue On Failure
         ...    Element Should Be Enabled    xpath=//div[contains(@class,"modal")]//div[contains(@class,"pf-c-select")]
-        FOR    ${workbench_title}    IN    @{connected_workbench}
-            Set Connection Between Data Connection And Workbench    ${workbench_title}
-            Run Keyword And Continue On Failure     Element Should Be Enabled    ${S3_DC_ADD_BTN}
-        END
+        Set Connection Between Data Connection And Workbench    ${workbench_title}
```

Comment:
why removing the loop? For the moment workbenches supports only one data connection, but it may support more in the next development phase. It used to support multiple data connection for workbench but there was an issue with the feature and it's been limited to 1 for now